### PR TITLE
Skeleton for backend and frontend communication

### PR DESCRIPTION
**Pull request by VincentPheng**
Fixed CSS.

### CODE REVIEW

File: frontend/src/components/Signup.js [hunk 1]
```diff
@@ -5,50 +5,85 @@ class Signup extends Component {
...
+                        <br/>
+                        <input type="password" id="password-confirm" placeholder="Confirm Password" size="50" required/>
+                        <br/>
+                        <section>
+                            {this.state.userType === "volunteer" &&
+                                <div id="volunteer-type">
+                                    <input type="radio" id="driver" name="volunteer" value="driver" onChange={this.changeVolunteerType}/>
+                                    <label for="driver">Driver</label>
+                                    <input type="radio" id="other" name="volunteer" value="other" onChange={this.changeVolunteerType}/>
+                                    <label for="other">Other/Cook</label>
```
Comment (HariCodesThings) [inline frontend/src/components/Signup.js]: Change to "Kitchen Volunteer" instead of "Other/Cook"

File: frontend/src/components/Signup.js [hunk 2]
```diff
@@ -5,50 +5,155 @@ class Signup extends Component {
...
+        this.setState({personalData: personalData});
+    }
+
+    changeUserType = (event) => {
+        this.setState({userType: event.target.value});
+    }
+
+    changeVolunteerType = (event) => {
+        let volunteerData = this.state.volunteerData;
+        if (event.target.value === "driver") {
```
Comment (EmilyGavrilenko) [inline frontend/src/components/Signup.js:41]: A user should have the option to be both a driver and a kitchen staff. It's not exclusive

File: frontend/.eslintcache [hunk 3]
```diff
@@ -1 +1 @@
...
...H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\components\\Home.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect-master\\frontend\\src\\components\\Signup.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\index.js",[],["60","61"],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\App.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\components\\NavBar.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\components\\Login.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\components\\Home.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\components\\Signup.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\components\\FrontBanner.js",[],"C:\\Users\\Vincent\\Documents\\H4I Bootcamp\\meals-that-connect\\frontend\\src\\components\\Columns.js",[],{"ruleId":"62","replacedBy":"63"},{"ruleId":"64","replacedBy":"65"},"no-native-reassign",["66"],"no-negated-in-lhs",["67"],"no-global-assign","no-unsafe-negation"]
```
Comment (EmilyGavrilenko) [inline frontend/.eslintcache]: .eslintcache should be in your gitignore

**Review by EmilyGavrilenko — APPROVED** [review]
Comment: a volunteer should have the option to be both a driver and a kitchen staff.

**Pull request merged**